Buildscript dependency `com.android.tools.build:gradle:3.6.0` requires the `google` maven repository

Your plugin has a dependency on `com.android.tools.build:gradle:3.6.0`, meaning the the `google` repository must be declared.

---
To reproduce the issue:

Create a new java project with gradle init:

```
gradle init

Select type of project to generate: 2 application
Select implementation language: 3 Java
Select build script DSL: 1 Groovy
Select test framework: 4 JUnit Jupiter

```

Edit the build script as indicated on the readme (see `ADDED` sections) :

```
/*
 * This file was generated by the Gradle 'init' task.
 *
 * This generated file contains a sample Java project to get you started.
 * For more details take a look at the Java Quickstart chapter in the Gradle
 * User Manual available at https://docs.gradle.org/5.5.1/userguide/tutorial_java_projects.html
 */

// ADDED 1
buildscript {
  repositories {
    jcenter()
  }

  dependencies {
    classpath 'com.jaredsburrows:gradle-license-plugin:0.8.70'
  }
}
// END ADDED 1

plugins {
    // Apply the java plugin to add support for Java
    id 'java'

    // Apply the application plugin to add support for building a CLI application
    id 'application'
}

// ADDED 2
apply plugin: 'com.jaredsburrows.license'
// END ADDED 2

repositories {
    // Use jcenter for resolving dependencies.
    // You can declare any Maven/Ivy/file repository here.
    jcenter()
}

dependencies {
    // This dependency is used by the application.
    implementation 'com.google.guava:guava:27.1-jre'

    // Use JUnit Jupiter API for testing.
    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.4.2'

    // Use JUnit Jupiter Engine for testing.
    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.4.2'
}

application {
    // Define the main class for the application
    mainClassName = 'gradletmp2.App'
}

test {
    // Use junit platform for unit tests
    useJUnitPlatform()
}
```

Run gradle build:

```
gradle licenseReport
```

You get:

```

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'gradletmp2'.
> Could not resolve all artifacts for configuration ':classpath'.
   > Could not find com.android.tools.build:gradle:3.6.0.
     Searched in the following locations:
       - https://jcenter.bintray.com/com/android/tools/build/gradle/3.6.0/gradle-3.6.0.pom
       - https://jcenter.bintray.com/com/android/tools/build/gradle/3.6.0/gradle-3.6.0.jar
       - https://plugins.gradle.org/m2/com/android/tools/build/gradle/3.6.0/gradle-3.6.0.pom
     Required by:
         project : > com.jaredsburrows:gradle-license-plugin:0.8.70

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s

```

----
Solution:

Add `google()` as repository in the `buildscript` section:

```
buildscript {
  repositories {
    jcenter()
    google()
  }

  dependencies {
    classpath 'com.jaredsburrows:gradle-license-plugin:0.8.70'
  }
}
```

----

The readme could be improved.

--- a/README.md
+++ b/README.md
@@ -19,6 +19,7 @@ Also, for Android projects the license HTML file will be copied to `<project>/sr
 buildscript {
   repositories {
     jcenter()
+    google()
   }
 
   dependencies {
@@ -36,6 +37,7 @@ Release versions are available in the [JFrog Bintray repository](https://jcenter
 buildscript {
   repositories {
     maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
+    google()
   }
 
   dependencies {

